src/modules/mim/daemonconfiguration.json

```diff
@@ -0,0 +1,143 @@
+                          "enumValue": 4
+                        }
+                      ]
+                    }
+                  },
+                  {
+                    "name": "status",
+                    "schema": {
+                      "type": "enum",
+                      "valueSchema": "integer",
+                      "enumValues": [
+                        {
+                          "name": "unknown",
+                          "enumValue": 0
+                        },
+                        {
+                          "name": "enabled",
+                          "enumValue": 1
+                        },
+                        {
+                          "name": "enabledRuntime",
+                          "enumValue": 2
+                        },
+                        {
+                          "name": "disabled",
+                          "enumValue": 3
+                        },
+                        {
+                          "name": "static",
+                          "enumValue": 4
+                        },
+                        {
+                          "name": "indirect",
+                          "enumValue": 5
+                        }
+                      ]
+                    }
+                  }
+                ]
+              }
+            }
+          },
+          {
+            "name": "daemonCommands",
+            "type": "mimObject",
+            "desired": true,
+            "schema": {
+              "type": "array",
+              "elementSchema": {
+                "type": "object",
+                "fields": [
+                  {
+                    "name": "name",
+                    "schema": "string"
+                  },
+                  {
+                    "name": "state",
+                    "schema": {
+                      "type": "enum",
+                      "valueSchema": "integer",
+                      "enumValues": [
+                        {
+                          "name": "run",
+                          "enumValue": 0
+                        },
+                        {
+                          "name": "stop",
+                          "enumValue": 1
+                        }
+                      ]
+                    }
+                  },
+                  {
+                    "name": "status",
+                    "schema": {
+                      "type": "enum",
+                      "valueSchema": "integer",
+                      "enumValues": [
+                        {
+                          "name": "enable",
+                          "enumValue": 0
+                        },
+                        {
+                          "name": "disable",
+                          "enumValue": 1
+                        }
+                      ]
+                    }
+                  }
+                ]
+              }
+            }
+          }
+        ]
```

Comment:
How does the desired object specify desired state and desired status for each existing daemon? 

Cannot expect the order in the both arrays to be same at all times.  The desired object may need the names at least.